cmake/caches/Windows-arm64.cmake

```diff
@@ -23,6 +23,7 @@ set(ENABLE_X86_RELAX_RELOCATIONS YES CACHE BOOL "")
 # NOTE(compnerd) we can hardcode the default target triple since the cache files
 # are target dependent.
 set(LLVM_DEFAULT_TARGET_TRIPLE aarch64-unknown-windows-msvc CACHE STRING "")
+set(SWIFT_HOST_VARIANT_ARCH aarch64 CACHE STRING "")
```

Comment:
Hmm, why was this suddenly needed?

Reply:
Consider the following case: we're running x86-64 native prompt on windows/arm. Here we will end with awkward situation since LLVM will be compiled for x86-64, but parts of swift – for windows/aarch64 (as swift infers `HOST_VARIANT` regardless of LLVM default target triple, etc.) causing link failures. This is a way to ensure everything is aligned.

Reply:
Ah, the caches are setup for split toolchain/runtime builds.  I think that is the cause of that confusion.  Perhaps we should comment that instead?  The only time this actually matters is if you are running the tests for development, where you are unlikely to be cross-compiling.

Reply:
Well, caches – yes. But `WindowsBuild.md` suggests use of compiler-side cache file, but builds both compiler and runtime.

Reply:
Ah, yeah, we should update that for the split model.  I worry that this might have an adverse effect on the official builds is all as the cache files are used by that.